### ISSUE TITLE
Can't update on Linux

### ISSUE THREAD

**Issue by hellofaduck** [Ylianst/MeshCentral#1598]
Using meshcentral about 6-7 month and have no problem with updates, but today something goes wrong. I can't understand why node trying to install a something like windows-beta on my linux machine? :)) I use this update script all the time and it works like swiss clock.
**My update script:**
```
#!/bin/bash
service meshcentral stop
cd /opt/meshcentral
npm install meshcentral
chown -R meshcentral:meshcentral /opt/meshcentral
service meshcentral start

```
**npn exited with this error:**
 ```
sudo ./mesh_update.sh
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for node-windows@1.0.0-beta.1: wanted {"os":"win32","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    win32
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/captain/.npm/_logs/2020-07-11T07_09_58_960Z-debug.log

```
**Error from debug.log:**
```
90 verbose stack Error: Unsupported platform for node-windows@1.0.0-beta.1: wanted {"name":"node-windows","version":"1.0.0-beta.1","os":["win32"],"dependencies":{"optimist":"~0.6.0","xml":"0.0.12"},"optionalDependencies":{},"peerDependenciesMeta":{},"devDependencies":{},"bundleDependencies":false,"peerDependencies":{},"deprecated":false,"_resolved":"https://registry.npmjs.org/node-windows/-/node-windows-1.0.0-beta.1.tgz","_integrity":"sha512-wWAM7zRvu4ZrBGsEI+7oGcoGr5jEeHgxzdx4DNEFddK/gh1yOsy782nCcShjOZ4Kffy8zDfmCw0Femxvib/5Rw==","_shasum":"a3442389c2981a5f504777172a50c912c5df2e6b","_shrinkwrap":null,"_id":"node-windows@1.0.0-beta.1","_from":"node-windows@^1.0.0-beta.1","_requested":{"type":"range","registry":true,"raw":"node-windows@^1.0.0-beta.1","name":"node-windows","escapedName":"node-windows","rawSpec":"^1.0.0-beta.1","saveSpec":null,"fetchSpec":"^1.0.0-beta.1"},"_spec":"node-windows@^1.0.0-beta.1","_where":"/opt/meshcentral/node_modules/meshcentral"} (current: {"os":"linux","cpu":"x64"})
90 verbose stack     at checkPlatform (/usr/lib/node_modules/npm/node_modules/npm-install-checks/index.js:46:14)
90 verbose stack     at thenWarnEngineIssues (/usr/lib/node_modules/npm/lib/install/validate-args.js:63:5)
90 verbose stack     at a (/usr/lib/node_modules/npm/node_modules/iferr/iferr.js:3:64)
90 verbose stack     at checkEngine (/usr/lib/node_modules/npm/node_modules/npm-install-checks/index.js:11:20)
90 verbose stack     at module.exports.isInstallable (/usr/lib/node_modules/npm/lib/install/validate-args.js:60:3)
90 verbose stack     at resolveWithNewModule (/usr/lib/node_modules/npm/lib/install/deps.js:677:10)
90 verbose stack     at /usr/lib/node_modules/npm/lib/install/deps.js:600:7
90 verbose stack     at a (/usr/lib/node_modules/npm/node_modules/iferr/iferr.js:3:64)
90 verbose stack     at process.nextTick (/usr/lib/node_modules/npm/node_modules/call-limit/call-limit.js:28:43)
90 verbose stack     at process._tickCallback (internal/process/next_tick.js:61:11)
91 verbose pkgid node-windows@1.0.0-beta.1
92 verbose cwd /opt/meshcentral
93 verbose Linux 4.15.0-106-generic
94 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "meshcentral"
95 verbose node v10.18.0
96 verbose npm  v6.14.5
97 error code EBADPLATFORM
98 error notsup Unsupported platform for node-windows@1.0.0-beta.1: wanted {"os":"win32","arch":"any"} (current: {"os":"linux","arch":"x64"})
99 error notsup Valid OS:    win32
99 error notsup Valid Arch:  any
99 error notsup Actual OS:   linux
99 error notsup Actual Arch: x64
100 verbose exit [ 1, true ]
```

**Comment by Ylianst** [comment]
Working on it!

**Comment by Ylianst** [comment]
Fixed, MeshCentral v0.5.85 is published! Let me know if it works.

**Comment by hellofaduck** [comment]
Oh yeah! Works great, thank you for your work!
![изображение](https://user-images.githubusercontent.com/49034105/87223600-0a5ba180-c387-11ea-8275-0d55270aa40a.png)
